### ISSUE TITLE
/kit reports error

### ISSUE THREAD

**Issue by jack444410** [Zrips/CMI#6932]
### Description of Issue

when I was trying to enter /kit, it reported "An internal error occurred while attempting to perform this command"

### Version Information

```shell
[17:54:41 INFO]: --------------------------------------------------
[17:54:41 INFO]: CMI插件版本: 9.1.4.7 BungeeCord  MySQL
[17:54:41 INFO]: CMILib: 1.1.2.4 -> 1.1.2.5
[17:54:41 INFO]: Server: Paper(334) 1.18.2-R0.1-SNAPSHOT
[17:54:41 INFO]: CMI economy: 停用 Vault: 1.7.3-b CMI Chat: 停用 
[17:54:41 INFO]: Modules -> 55 enabled 1 disabled: votifier
[17:54:41 INFO]: --------------------------------------------------
```

### Errors

```shell
[17:51:40 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'cmi' in plugin CMI v9.1.4.7
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R2.CraftServer.dispatchCommand(CraftServer.java:906) ~[paper-1.18.2.jar:git-Paper-334]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleCommand(ServerGamePacketListenerImpl.java:2306) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:2117) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:2098) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:46) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundChatPacket.a(ServerboundChatPacket.java:6) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:51) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.18.2.jar:git-Paper-334]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1399) ~[paper-1.18.2.jar:git-Paper-334]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:188) ~[paper-1.18.2.jar:git-Paper-334]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1376) ~[paper-1.18.2.jar:git-Paper-334]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1369) ~[paper-1.18.2.jar:git-Paper-334]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1347) ~[paper-1.18.2.jar:git-Paper-334]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1229) ~[paper-1.18.2.jar:git-Paper-334]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315) ~[paper-1.18.2.jar:git-Paper-334]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NoSuchMethodError: 'void net.Zrips.CMILib.GUI.CMIGui.fillInEdges()'
        at com.Zrips.CMI.Modules.Kits.KitsManager.listPlayersKitsInComplexGUI(KitsManager.java:1107) ~[CMI9.1.4.7.jar:?]
        at com.Zrips.CMI.Modules.Kits.KitsManager.listPlayersKitsInGUI(KitsManager.java:1214) ~[CMI9.1.4.7.jar:?]
        at com.Zrips.CMI.commands.list.kit.perform(kit.java:120) ~[CMI9.1.4.7.jar:?]
        at com.Zrips.CMI.commands.CommandsHandler.onCommand(CommandsHandler.java:399) ~[CMI9.1.4.7.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        ... 21 more
```

### Relevant Config Sections

```yaml
Kits:
  # When set to true, kit list will be shown in GUI instead of chat list
  GUI: true
  # When set to true, kit selection gui empty fields will get filled with definet item
  FillEmptyFields: true
  Buttons:
    Cooldown: Watch
    Usages: STONE_PLATE
    Money: GOLD_INGOT
    Exp: EXP_BOTTLE
    Desc: WOOL:13
    Back: Fence
  Complex:
    CloseButton:
      Use: true
      Slot: 9
      Material: head:eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNmNjYmY5ODgzZGQzNTlmZGYyMzg1YzkwYTQ1OWQ3Mzc3NjUzODJlYzQxMTdiMDQ4OTVhYzRkYzRiNjBmYyJ9fX0=
      Commands:
      - closeinv!
    InfoButton:
      # Extra button to be used in case you want to provide any aditional information when clicking on it
      Use: false
      Slot: 1
      Material: head:eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNDZiYTYzMzQ0ZjQ5ZGQxYzRmNTQ4OGU5MjZiZjNkOWUyYjI5OTE2YTZjNTBkNjEwYmI0MGE1MjczZGM4YzgyIn19fQ==
      Commands:
      - closeinv!
```

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by mrfloris** [comment]
Step one would be to backup your server and remove the old cmilib jar(s) and get the latest cmilib jar 1.1.2.5 and try again. Let us know if the issue persists after the cmilib update.

**Comment by jack444410** [comment]
oh.. it fixed after I remvoed the 1.1.2.4 jar file.
sorry for misreporting.
